src/reference/90-Developers-Guide/09-Launcher/03-Launcher-Configuration.md

```diff
@@ -40,7 +40,7 @@ The default configuration file for sbt as an application looks like:
 
 [repositories]
   local
-  typesafe-ivy-releases: http://repo.typesafe.com/typesafe/ivy-releases/, [organization]/[module]/[revision]/[type]s/[artifact](-[classifier]).[ext], bootOnly
+  typesafe-ivy-releases: http://repo.typesafe.com/typesafe/ivy-releases/, [organization]/[module]/(scala_[scalaVersion]/)(sbt_[sbtVersion]/)[revision]/[type]s/[artifact](-[classifier]).[ext], bootOnly
```

Comment:
But typesafe-ivy-releases hosts sbt 0.13 artifacts like this https://repo.typesafe.com/typesafe/ivy-releases/org.scala-sbt/sbt/0.13.18/ so I am not sure if this is necessary.

Reply:
Ah ok, didn't realize that it was that they were for. I just used this line as a template for setting up our own repos, which then failed. The other option I see would be to document `scalaVersion` and `sbtVersion`.

Reply:
You need those to resolve extra properties set on sbt plugins. So https://github.com/sbt/website/blob/develop/src/reference/02-DetailTopics/03-Dependency-Management/04-Proxy-Repositories.md does say:

```
   my-ivy-proxy-releases: http://repo.company.com/ivy-releases/, [organization]/[module]/(scala_[scalaVersion]/)(sbt_[sbtVersion]/)[revision]/[type]s/[artifact](-[classifier]).[ext]
```

Reply:
Maybe it's better to add:

```
  # not actually in the default launcher, but added for documentation
  my-ivy-proxy-releases: http://repo.company.com/ivy-releases/, [organization]/[module]/(scala_[scalaVersion]/)(sbt_[sbtVersion]/)[revision]/[type]s/[artifact](-[classifier]).[ext]
  my-maven-proxy-releases: http://repo.company.com/maven-releases/
```

rather than changing `typesafe-ivy-releases` from what's actually being shipped.